src/Sylius/Bundle/AdminBundle/Resources/config/app/sylius/resources/inventory.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+//
+///*
+// * This file is part of the Sylius package.
+// *
+// * (c) Sylius Sp. z o.o.
+// *
+// * For the full copyright and license information, please view the LICENSE
+// * file that was distributed with this source code.
+// */
+//
+//declare(strict_types=1);
+//
+//use Sylius\Resource\Metadata\Index;
+//use Sylius\Resource\Metadata\Operations;
+//use Sylius\Resource\Metadata\ResourceMetadata;
+//
+//return (new ResourceMetadata())
+//    ->withRoutePrefix('/admin')
+//    ->withClass('%sylius.model.product_variant.class%')
+//    ->withSection('admin')
+//    ->withTemplatesDir("@SyliusAdmin\\shared\\crud")
+//    ->withOperations(new Operations(operations: [
+//        new Index(
+//            path: 'inventory',
+//            shortName: 'inventory',
+//            name: 'sylius_admin_inventory_index',
```

Comment:
We need a fix on Resource package to allow changing the whole route name with operation name.

Reply:
I think we should use routeName property